### ISSUE TITLE
Hive Mind Mode: share informations about other bots to descramble Ogar Seed values

### ISSUE THREAD

**Issue by davidmann4** [davidmann4/ogar-feeder-bot#200]
- add the mode to config.js
- add the new behavior to the feeder.js
- emit a packet when a bot has found a corner value and spots other bots
- add the logic to calaculate own seed value based on the seed value from other bots
- distribute the different offsets via the server

**Comment by issy123** [comment]
This wont work.

First of all every bots its offset is different, with up to an offset more than 4,000
So you cant say bot 1 hit the corner now other bots now it too, only if it is within the bots viewport so he could calculate how far the bot is from the corner to normalize coordinations.

Okay say now you got the bots with the correct distance from the corner, aka normalized coordinations.

They know what is left and what is right but still the player does not know where he is.

It will only work if the player too has hit the corner, and i dont hit the corner each gameplay..

**Comment by davidmann4** [comment]
If one bot already knows his offset it can calculate the offset of all other elements - this includes other bots.

Its like they pass by the other bots / players and yell "sup, your offset is 444,333 (based on 0) they can use this information to compute their own offset

**Comment by issy123** [comment]
Yes but they need to yell at the player too, if the player does not know where he is and bots do know, then bots do not know where to go

**Comment by mbm3** [comment]
@issy123 The player can get offsets from the nearby bots. It's like a network that splits up everywhere.

**Comment by davidmann4** [comment]
I see no reason why this should not work.

**Comment by xa** [comment]
If one bot can see target cell it can be marked as 'found target' and other bots can follow him too. It could create network of bots that are following the same target.

http://i.imgur.com/4Q9RxbE.png

**Comment by hassanila97** [comment]
![](http://i.imgur.com/4Q9RxbE.png)

**Comment by WJKH** [comment]
You're not good at paint guys :D

**Comment by hassanila97** [comment]
Just do it

**Comment by mayed505** [comment]
David Please it would be nice to add @hassanila97 's idea.
p.s i wonder how raga.pw got mouse movement working.
